src/mixins/filing-template-mixin.ts

```diff
@@ -691,6 +692,83 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     this.storeStaffPayment(filing)
   }
 
+  /**
+   * Parses a draft special resolution filing into the store.
+   * @param filing the special resolution filing
+   * @param entitySnapshot the latest entity snapshot
+   */
+  parseSpecialResolutionnFiling (filing: SpecialResolutionFilingIF, entitySnapshot: EntitySnapshotIF): void {
+    // store Entity Snapshot
+    this.setEntitySnapshot(entitySnapshot)
+
+    // store Entity Type
+    this.setEntityType(filing.alteration.business?.legalType || entitySnapshot.businessInfo.legalType)
+
+    // store Business Information
+    this.setBusinessInformation({
+      ...filing.business,
+      ...filing.alteration.business
+    })
+
+    // store Name Request data
+    this.setNameRequest(
+      filing.alteration.nameRequest ||
+      {
+        legalType: entitySnapshot.businessInfo.legalType,
+        legalName: entitySnapshot.businessInfo.legalName,
+        nrNumber: entitySnapshot.businessInfo.nrNumber
+      }
+    )
+
+    // store Provisions Removed
+    this.setProvisionsRemoved(filing.alteration.provisionsRemoved)
+
+    // store Office Addresses **from snapshot** (because we don't change office addresses in an alteration)
+    this.setOfficeAddresses(entitySnapshot.addresses)
+
+    // store People And Roles **from snapshot** (because we don't change people and roles in an alteration)
+    this.setPeopleAndRoles(entitySnapshot.orgPersons)
+
+    // store Business Contact
+    this.setBusinessContact(entitySnapshot.authInfo.contact)
+
+    // store Share Classes and Resolution Dates
+    this.setShareClasses(
```

Comment:
You don't need this for CPs.

I'd prefer to not review your PR each time you add or clean up functionality. I agree with you that small PRs are nice but if we merged this code then there would be tech debt to clean it up later and I don't know how we'd keep track of that, so I think it's worth some effort up front to only merge changes that won't need cleanup later.

Reply:
^^ less tech debt not more, please

Reply:
yeah agree, Since this is a basic setup, I tried to keep most of the components there. WE have tickets to work on each component. This part will come on click on save and resume I suppose. I removed most of the component, i feel like not using / not going to use in future